src/item.cpp

```diff
@@ -9401,29 +9159,19 @@ bool item::process_blackpowder_fouling( player *carrier )
 }
 
 bool item::process( player *carrier, const tripoint &pos, bool activate, float insulation,
-                    temperature_flag flag )
+                    temperature_flag flag, float spoil_multiplier )
 {
-    const bool preserves = type->container && type->container->preserves;
-    std::vector<item *> removed_items;
-    visit_items( [&]( item * it ) {
-        if( preserves ) {
-            // Simulate that the item has already "rotten" up to last_rot_check, but as item::rot
-            // is not changed, the item is still fresh.
-            it->last_rot_check = calendar::turn;
-        }
-        if( it->process_internal( carrier, pos, activate, type->insulation_factor * insulation, flag ) ) {
-            removed_items.push_back( it );
-        }
-        return VisitResponse::NEXT;
-    } );
-    for( item *it : removed_items ) {
-        remove_item( *it );
-    }
-    return !removed_items.empty();
+    contents.process( carrier, pos, activate, insulation, flag, spoil_multiplier );
```

Comment:
I believe this needs to be changed into something like this to apply the insulation and smoil multiplier of the container.

`contents.process( carrier, pos, activate, insulation*[Insulation of this container], flag, spoil_multiplier*[spoil multiplier of this container] );`